server/game/cards/characters/04/aerondamphair.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../../drawcard.js');
+
+class AeronDamphair extends DrawCard {
+
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardSaved: event => event.card.getType() === 'character'
+            },
+            choices: {
```

Comment:
ah, good point. And it also simplifies things, which is good :)

I briefly looked at the other PR (thanks for the pointer!) and AFAICT this PR already implements the event naming/granularity that @ystros wanted, as well as the duplicate saving part.

I'll check for other corner cases mentioned there before updating this PR again.